### PR TITLE
feat: add support ConfluxGetAddress for conflux network

### PR DESCRIPTION
**Pull request by lyxyx**

### CODE REVIEW

**Comment by rayston92** [comment]
@loatheb is this recommended way to merge codes? if so, workflows need to be approved and run.

cc @lyxyx

**Comment by revan-zhang** [comment]
Yes, this is the way for outside contributor, after code has been reviewed and approved, then the CI will approve and run.Currently this PR can convert to dfrat cc @lyxyx .

File: common/protob/messages.proto [hunk 1]
```diff
@@ -347,6 +347,12 @@ enum MessageType {
     MessageType_BinanceCancelMsg = 708 [(wire_in) = true];
     MessageType_BinanceSignedTx = 709 [(wire_out) = true];
 
+    // Conflux
+    MessageType_ConfluxGetAddress = 900 [(wire_in) = true];
```
Comment (lihuanhuan) [inline common/protob/messages.proto]: ID指定尽量避开trezor后续能使用到的区间

File: common/protob/messages-conflux.proto [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+/**
+ * Request: Ask device to verify message
+ * @start
+ * @next Success
+ * @next Failure
+ */
+ message ConfluxVerifyMessageCIP23 {
+    required bytes signature = 2;   // signature to verify
+    required bytes domain_hash = 3;     // domain_separator hash
+    required bytes message_hash = 34;     // struct_hash
```
Comment (somebodyLi) [inline common/protob/messages-conflux.proto]: typo?

File: core/src/apps/conflux/sign_tx.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    await paths.validate_path(ctx, keychain, msg.address_n)
+    node = keychain.derive(msg.address_n)
+
+    await require_confirm_tx(ctx, msg.to, int.from_bytes(msg.value, "big"))
+    if msg.data_length > 0:
+        await require_confirm_data(ctx, msg.data_initial_chunk, data_total)
+
+    msg.to = decode_hex_address(msg.to)
+
+    filled_transaction = {}
```
Comment (somebodyLi) [inline core/src/apps/conflux/sign_tx.py]: 这个临时变量可以不要？

File: python/src/trezorlib/conflux.py [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+        data_length = response.data_length
+        data, chunk = data[data_length:], data[:data_length]
+        response = client.call(messages.ConfluxTxAck(data_chunk=chunk))
+        assert isinstance(response, messages.ConfluxTxRequest)
+
+    assert response.signature_v is not None
+    assert response.signature_r is not None
+    assert response.signature_s is not None
+
+    print(response)
```
Comment (somebodyLi) [inline python/src/trezorlib/conflux.py]: ？？

File: common/protob/messages-conflux.proto (hunk 2 above)
Comment (lyxyx) [reply common/protob/messages-conflux.proto]: typo已改

File: core/src/apps/conflux/sign_tx.py (hunk 3 above)
Comment (lyxyx) [reply core/src/apps/conflux/sign_tx.py]: 已调整并测试了

**Review by somebodyLi — APPROVED** [review]

File: common/protob/messages.proto (hunk 1 above)
Comment (lyxyx) [reply common/protob/messages.proto]: 已调整并规范到文档中

**Pull request merged**